oisatgmi/interpolator.py

```diff
@@ -6,7 +6,7 @@
 from scipy import signal
 from scipy.interpolate.interpnd import _ndim_coords_from_arrays
 from scipy.spatial import cKDTree
-
+from scipy.io import savemat
```

Comment:
please remove this after you've tested something

Reply:
okay~